polaris/utils/types.py

```diff
@@ -115,11 +115,21 @@ def __str__(self):
         return self.slug
 
 
+class SupportedLicenseType(Enum):
```

Comment:
I'm not sure if we need an Enum here. I suggest we just define a new `TypeAlias` for a `Literal` to prevent confusion on whether the use the name or key of the enum.